src/codewhisperer/util/authUtil.ts

```diff
@@ -33,6 +33,10 @@ export const isValidCodeWhispererConnection = (conn: Connection): conn is Connec
         return isIamConnection(conn)
     }
 
+    if (isSageMaker()) {
```

Comment:
could put this in the condition on line 32, above

Reply:
it could, I was just trying to avoid coupling them if I could